lib/order.rb

```diff
@@ -0,0 +1,97 @@
+# allows use of csv file
+require 'csv'
+# allows access to customer.rb
+require_relative 'customer'
+
+# creates Order class
+class Order
+  # reads data from orders.csv file
+  @@order = CSV.read("data/orders.csv")
```

Comment:
I think that the name `@@order` doesn't make much sense-- plural `@@orders` may be better, or `@@data`. Then it makes more sense when you use `@@data.map` below